client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -350,15 +350,41 @@
                     superdesk.findActivities(intent, item).forEach(function (activity) {
                         if (activity.keyboardShortcut && workflowService.isActionAllowed(item, activity.action)) {
                             monitoring.bindedItems.push(activity.keyboardShortcut);
-
                             keyboardManager.bind(activity.keyboardShortcut, function () {
-                                activityService.start(activity, {data: {item: scope.selected}});
+                                if (activity._id === 'mark.item') {
+                                    bindMarkItemShortcut();
+                                } else {
+                                    activityService.start(activity, {data: {item: scope.selected}});
+                                }
+
                             }, {inputDisabled: true});
                         }
                     });
                 }
 
                 /*
+                 * Bind highlight dropdown action
+                 * Keyboard shortcut is defined with action
+                 *
+                 * @param {Object} item
+                 */
+                function bindMarkItemShortcut() {
+                    elem.find('.active .more-activity-toggle').click();
+                    var highlightDropdown = angular.element('.more-activity-menu.open .dropdown-noarrow');
+
+                    highlightDropdown.addClass('open');
+                    if (highlightDropdown.find('button').length > 0) {
+                        highlightDropdown.find('button')[0].focus();
+                        keyboardManager.push('up', function () {
+                            highlightDropdown.find('button:focus').parent('li').prev().children('button').focus();
+                        });
+                        keyboardManager.push('down', function () {
+                            highlightDropdown.find('button:focus').parent('li').next().children('button').focus();
```

Comment:
It is not pointing to the same element every time. `Find` function is looking for focused button, so if I put this out of `keyboardManager` function, it will pointing to the same button every time.